com.reprezen.swagedit.dependencies/META-INF/MANIFEST.MF

```diff
@@ -95,6 +96,6 @@ Export-Package: com.fasterxml.jackson.annotation,
  org.yaml.snakeyaml.serializer,
  org.yaml.snakeyaml.tokens,
  org.yaml.snakeyaml.util
-Require-Bundle: com.google.guava;bundle-version="21.0.0",
+Require-Bundle: com.google.guava;bundle-version="[14.0.0,22.0.0)",
```

Comment:
Can KZOE really work with this range of Guava releases? 

I know we have done some work to eliminate guava dependencies from our components. How much work remains to completely eliminate the Guava dependency from KZOE?

Reply:
It's not about KZOE, it's the schema validator that uses Guava. We need to have a version that is compatible with the Guava version used by Xtend. And this is the range that uses Xtend. See https://github.com/ModelSolv/RepreZen/pull/1871